prime-router/src/main/resources/db/migration/V22__add_look_tables.sql

```diff
@@ -20,6 +20,7 @@
   lookup_table_version_id BIGSERIAL PRIMARY KEY,
   table_name character varying(512) NOT NULL,
   table_version INTEGER NOT NULL CHECK (table_version >= 0),
+  table_Sha256 character varying(512) NULL,
```

Comment:
This is not the way Flyway works.  You need to create a brand new file, with the next version and write code to update the table.  See Flyways docs for more info.

Reply:
Done.  Thanks.